core/src/main/java/org/sonarsource/sonarlint/core/NodeJsHelper.java

```diff
@@ -61,8 +62,14 @@ public void detect(@Nullable Path configuredNodejsPath) {
     detectedNodePath = locateNode(configuredNodejsPath);
     if (detectedNodePath != null) {
       logOutput.log("Checking node version...", ClientLogOutput.Level.DEBUG);
-      var command = Command.create(detectedNodePath.toString()).addArgument("-v");
-      var nodeVersionStr = runSimpleCommand(command);
+      String nodeVersionStr;
+      var forcedNodeVersion = System.getProperty("sonarlint.internal.nodejs.forcedVersion");
+      if(Objects.nonNull(forcedNodeVersion)) {
```

Comment:
```suggestion
      if (Objects.nonNull(forcedNodeVersion)) {
```

Reply:
Any reason to not simply use `forcedNodeVersion != null` ?

Reply:
I wanted to look educated. But it turned out that using Objects.nonNull is useful when a predicate is needed...I will simplify it.